pagopa/pn-frontend
# fix/PN-9620: fix spacing in modal PF/PA/PG for tablet

## List of changes proposed in this pull request
- removed isMobile hook where not needed
- added padding property in DialogTitle, DialogContent and DialogAction with breakpoints
- fixed direction and size of button in DialogAction


## How to test
Login as PF/PG/PA and check in tablet:
- padding of entire modal: 32px
- bottom distance between title and body: 16px
- bottom distance between title actions: 32px
- bottom distance between title elements in body: 16px
